terraform/s3.tf

```diff
@@ -62,6 +59,29 @@ resource "aws_s3_bucket" "logs" {
   }
 }
 
+resource "aws_s3_bucket_acl" "logs_acl" {
```

Comment:
this is an opinion on naming, you don't need tio include the resource type in the name, calling all the s3 resources for teh bucket "logs" just "logs" (and not logs_acl) is fine, unique enough and will help if you have to do state operations at some later date.